main.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from db.engine import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors", response_model=list[schemas.Author])
+def read_authors(
+    skip: int = 0, limit: int = 5, db: Session = Depends(get_db)
+):
+    return crud.get_all_authors(db=db, skip=skip, limit=limit)
+
+
+@app.post("/authors", response_model=schemas.AuthorCreate)
```

Comment:
```suggestion
@app.post("/authors/", response_model=schemas.AuthorCreate)
```

don't forget about this